### DISCUSSION TITLE
Why destination-doris removed?

### DISCUSSION THREAD

**Discussion by dhl777** [airbytehq/airbyte#43303] [Connector Questions]
### Topic

destination-doris

### Relevant information

"destination-doris" is a very useful connector, but I found that the latest version of Airbyte has removed it. What is the reason? Will it be added back in the subsequent version?

**Comment by Maaciekk** [comment]
any news regarding Destination-Doris?

**Comment by techieshark** [comment]
Docs also say it is 'archived' - https://docs.airbyte.com/integrations/destinations/doris 

And 'archived' is documented here: https://docs.airbyte.com/integrations/connector-support-levels/#archived

> Archived connectors will not receive any further updates or support from the Airbyte team. Archived connectors remain source-available in the [airbytehq/connector-archive](https://github.com/airbytehq/connector-archive) repository on GitHub.

> If you wish to take over the maintenance of an archived connector, please open a Github Discussion. For API Sources (python), updating the connector to the latest version of the [CDK](https://docs.airbyte.com/connector-development/cdk-python/) and ensuring that the connector successfully passes the [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference) is the start to the un-archiving process

**Reply by catpineapple** [reply]
hi, I'd like to reactivate this destination and plan to improve the performance and usability of the previous code implementation. I'm still developing in Java.  but Java Destination connector CDK seems to be under development. If I were to start this project now, what should I do?
